stac_validator.py

```diff
@@ -47,43 +47,24 @@ def __init__(self, stac_file, version="master"):
         :param stac_file: file to validate
         :param version: github tag - defaults to master
         """
-        git_tags = requests.get("https://cdn.staclint.com/versions.json")
-        if git_tags.status_code == 200:
-            stac_versions = git_tags.json()["versions"]
-        else:
-            git_tags = requests.get(
-                "https://api.github.com/repos/radiantearth/stac-spec/tags"
-            ).json()
-            stac_versions = [tag["name"] for tag in git_tags]
-
-        # cover master as well
-        if version is None:
-            version = "master"
-        stac_versions += ["master"]
-
-        if version not in stac_versions:
-            raise stac_exceptions.VersionException(
-                f"{version} is not a valid STAC version. Valid Versions are: {stac_versions}"
-            )
-
```

Comment:
resolves #36, we don't really need to list the version, IMO this should be taken care when trying to fetch the specification

Reply:
grab it from the cache instead of the cdn?

Reply:
This part only gets the stac version to validate the version passed by the user. My point was more to remove this to let the program fail if the we don't find the stac files either on the cdn or in the cache

Reply:
Added a comment to #36. I think we can keep the error, but call it under the fetch_spec method.